core/templates/pages/practice-session-page/practice-session-page.component.ts

```diff
@@ -109,6 +109,7 @@ angular.module('oppia').component('practiceSessionPage', {
         ctrl.stringifiedSubtopicIds = (
           UrlService.getSelectedSubtopicsFromUrl());
         _fetchSkillDetails();
+        LoaderService.hideLoadingScreen();
```

Comment:
Could you please point me to where the loading screen is initialised / shown when the practice session page component is initialised?

Reply:
Also, how do we ensure that the loading screen is hidden only after all the async calls required for the page to be interactable have completed (note that `_fetchSkillDetails` is async)?

Reply:
Ignore the first comment but please respond to the second. Thanks!

Reply:
You're right, I missed this! Moved the `hideLoadingScreen` call inside the fetch request's .then callback.